categorical_map/categorical_map.ipynb

```diff
@@ -0,0 +1,125 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This plot shows pollen health impact classes over the Alps on February 15, 2020 at 8 o'clock in the evening. It shows hourly values for Alder (Alnus) pollen and has been kindly provided by Simon Adamov. The field is generated by the COSMO-1E weather model, on the typical COSMO-1E domain over the greater alpine area. For better understanding, the numerical pollen concentrations are converted into health impact classes. These classes are then presented to the public via animated pollen maps in the MeteoSwiss App."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
```

Comment:
Could we add the output to the repo? I think it is nice if people are browsing the repository and they dont need to run the notebook just see clearly what it is doing from the output. What do you think? On the downside it makes git diff much more chaotic when we include output from notebooks to git.

Reply:
Thanks I added it. I also think it has advantages and disadvantages but let's be consistent with the other notebooks.